[2025-05-28T18:15:13.255+0100] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-28T18:15:13.479+0100] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_reviews_etl_and_analysis_pipeline.dbt_run_decisionnal scheduled__2025-04-13T00:00:00+00:00 [queued]>
[2025-05-28T18:15:13.541+0100] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_reviews_etl_and_analysis_pipeline.dbt_run_decisionnal scheduled__2025-04-13T00:00:00+00:00 [queued]>
[2025-05-28T18:15:13.541+0100] {taskinstance.py:2867} INFO - Starting attempt 2 of 2
[2025-05-28T18:15:13.691+0100] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): dbt_run_decisionnal> on 2025-04-13 00:00:00+00:00
[2025-05-28T18:15:13.721+0100] {standard_task_runner.py:72} INFO - Started process 9752 to run task
[2025-05-28T18:15:13.786+0100] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'bank_reviews_etl_and_analysis_pipeline', 'dbt_run_decisionnal', 'scheduled__2025-04-13T00:00:00+00:00', '--job-id', '2519', '--raw', '--subdir', 'DAGS_FOLDER/dag_load_reviews.py', '--cfg-path', '/tmp/tmppgdfsgtu']
[2025-05-28T18:15:13.789+0100] {standard_task_runner.py:105} INFO - Job 2519: Subtask dbt_run_decisionnal
[2025-05-28T18:15:14.085+0100] {task_command.py:467} INFO - Running <TaskInstance: bank_reviews_etl_and_analysis_pipeline.dbt_run_decisionnal scheduled__2025-04-13T00:00:00+00:00 [running]> on host DESKTOP-IM7V4RI.
[2025-05-28T18:15:14.742+0100] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohammed' AIRFLOW_CTX_DAG_ID='bank_reviews_etl_and_analysis_pipeline' AIRFLOW_CTX_TASK_ID='dbt_run_decisionnal' AIRFLOW_CTX_EXECUTION_DATE='2025-04-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-13T00:00:00+00:00'
[2025-05-28T18:15:14.760+0100] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-28T18:15:14.761+0100] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-28T18:15:14.762+0100] {logging_mixin.py:190} INFO - Current task name:dbt_run_decisionnal state:running start_date:2025-05-28 17:15:13.480188+00:00
[2025-05-28T18:15:14.762+0100] {logging_mixin.py:190} INFO - Dag name:bank_reviews_etl_and_analysis_pipeline and current dag run status:running
[2025-05-28T18:15:14.762+0100] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-28T18:15:14.775+0100] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-05-28T18:15:14.779+0100] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/mohammed/.dbt/dbt_projects/bank_reviews_decisionnal && dbt run --full-refresh']
[2025-05-28T18:15:14.892+0100] {subprocess.py:99} INFO - Output:
[2025-05-28T18:15:24.596+0100] {subprocess.py:106} INFO - [0m17:15:24  Running with dbt=1.9.3
[2025-05-28T18:15:25.272+0100] {subprocess.py:106} INFO - [0m17:15:25  Registered adapter: postgres=1.9.0
[2025-05-28T18:15:25.764+0100] {subprocess.py:106} INFO - [0m17:15:25  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-05-28T18:15:25.764+0100] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-05-28T18:15:25.764+0100] {subprocess.py:106} INFO - - models.bank_reviews_decisionnal.decisionnal
[2025-05-28T18:15:25.916+0100] {subprocess.py:106} INFO - [0m17:15:25  Found 5 models, 2 sources, 547 macros
[2025-05-28T18:15:25.919+0100] {subprocess.py:106} INFO - [0m17:15:25
[2025-05-28T18:15:25.919+0100] {subprocess.py:106} INFO - [0m17:15:25  Concurrency: 4 threads (target='dev')
[2025-05-28T18:15:25.919+0100] {subprocess.py:106} INFO - [0m17:15:25
[2025-05-28T18:15:26.080+0100] {subprocess.py:106} INFO - [0m17:15:26  1 of 5 START sql table model decisionnal.dim_bank .............................. [RUN]
[2025-05-28T18:15:26.083+0100] {subprocess.py:106} INFO - [0m17:15:26  2 of 5 START sql table model decisionnal.dim_branch ............................ [RUN]
[2025-05-28T18:15:26.084+0100] {subprocess.py:106} INFO - [0m17:15:26  3 of 5 START sql table model decisionnal.dim_location .......................... [RUN]
[2025-05-28T18:15:26.085+0100] {subprocess.py:106} INFO - [0m17:15:26  4 of 5 START sql table model decisionnal.dim_sentiment ......................... [RUN]
[2025-05-28T18:15:26.622+0100] {subprocess.py:106} INFO - [0m17:15:26  2 of 5 OK created sql table model decisionnal.dim_branch ....................... [[32mSELECT 298[0m in 0.53s]
[2025-05-28T18:15:26.630+0100] {subprocess.py:106} INFO - [0m17:15:26  1 of 5 OK created sql table model decisionnal.dim_bank ......................... [[32mSELECT 299[0m in 0.54s]
[2025-05-28T18:15:26.641+0100] {subprocess.py:106} INFO - [0m17:15:26  3 of 5 OK created sql table model decisionnal.dim_location ..................... [[32mSELECT 298[0m in 0.55s]
[2025-05-28T18:15:26.642+0100] {subprocess.py:106} INFO - [0m17:15:26  4 of 5 OK created sql table model decisionnal.dim_sentiment .................... [[32mSELECT 3[0m in 0.55s]
[2025-05-28T18:15:26.649+0100] {subprocess.py:106} INFO - [0m17:15:26  5 of 5 START sql table model decisionnal.fact_reviews .......................... [RUN]
[2025-05-28T18:15:26.868+0100] {subprocess.py:106} INFO - [0m17:15:26  5 of 5 OK created sql table model decisionnal.fact_reviews ..................... [[32mSELECT 2591[0m in 0.21s]
[2025-05-28T18:15:26.907+0100] {subprocess.py:106} INFO - [0m17:15:26
[2025-05-28T18:15:26.908+0100] {subprocess.py:106} INFO - [0m17:15:26  Finished running 5 table models in 0 hours 0 minutes and 0.99 seconds (0.99s).
[2025-05-28T18:15:26.962+0100] {subprocess.py:106} INFO - [0m17:15:26
[2025-05-28T18:15:26.963+0100] {subprocess.py:106} INFO - [0m17:15:26  [32mCompleted successfully[0m
[2025-05-28T18:15:26.963+0100] {subprocess.py:106} INFO - [0m17:15:26
[2025-05-28T18:15:26.964+0100] {subprocess.py:106} INFO - [0m17:15:26  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 TOTAL=5
[2025-05-28T18:15:28.068+0100] {subprocess.py:110} INFO - Command exited with return code 0
[2025-05-28T18:15:28.126+0100] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-28T18:15:28.126+0100] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=bank_reviews_etl_and_analysis_pipeline, task_id=dbt_run_decisionnal, run_id=scheduled__2025-04-13T00:00:00+00:00, execution_date=20250413T000000, start_date=20250528T171513, end_date=20250528T171528
[2025-05-28T18:15:28.185+0100] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-28T18:15:28.186+0100] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-28T18:15:28.186+0100] {logging_mixin.py:190} INFO - Dag name:bank_reviews_etl_and_analysis_pipeline queued_at:2025-05-28 17:08:58.166535+00:00
[2025-05-28T18:15:28.186+0100] {logging_mixin.py:190} INFO - Task hostname:DESKTOP-IM7V4RI. operator:BashOperator
[2025-05-28T18:15:28.197+0100] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-28T18:15:28.222+0100] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-28T18:15:28.227+0100] {local_task_job_runner.py:245} INFO - ::endgroup::
